README.md

```diff
@@ -121,6 +121,10 @@ export {
 }
 ```
 
+- `esModuleExternals {boolean}`: A boolean that determines whether esm-webpack-plugin will add the `__esModule` property to all imported externals. This can be helpful for improving interop between CJS and ESM modules, since webpack treats modules with the `__esModule` property differently than modules without them. Defaults to `true`.
```

Comment:
I'll add a note here after merging indicating that this option only applies to when `moduleExternals` is set to `true`